dapp/src/routes/Search/Search.svelte

```diff
@@ -24,11 +24,39 @@
     localNetwork = $network;
   });
 
+  const handleKeydown = (event) => {
+    if (event.key == 'Enter') {
+      event.preventDefault();
+      searchProfiles();
+    }
+  };
+
   const setSelectedNetwork = () => {
     network.set(localNetwork as NetworkType);
   };
+
+  const searchProfiles = async () => {
+    searching = true;
+    let searchingAddress = address;
+    if (address.includes('tez')) {
```

Comment:
I'd change the above two lines to:
```javascript
let searchingAddress = address.trim();
if (address.endsWith('.tez') {
```
So that you don't get bit by spaces in the search bar or a very unlucky PKH that happens to include "tez" in it's hash string which the `.` precludes